saos-webapp/src/main/webapp/WEB-INF/view/judgment/judgmentSearchForm.jsp

```diff
@@ -21,16 +21,15 @@
    			<saos:formFieldText path="signature" labelName="input-search-casenumber" labelText="judgmentSearch.formField.caseNumber" />
 
 		    <div class="form-group">
-			    <label for="datepicker_from" class="col-sm-2 control-label"><spring:message code="judgmentSearch.formField.date" />:</label>
+			    <label for="datepicker_from" class="col-sm-2 control-label"><spring:message code="judgmentSearch.formField.dateFrom" />:</label>
 			    <div class="col-sm-2">
-  		 			<spring:message code="judgmentSearch.formField.dateFrom" var="labelDateFrom" />
-			    	<form:input path="dateFrom" class="form-control" id="datepicker_from" placeholder="${labelDateFrom}" />
-			       
+  		 			<spring:message code="judgmentSearch.formField.datePlaceholder" var="labelDateFrom" />
```

Comment:
There is no need to fetch this label twice. Also name of var is a little misleading. Something like `datePlaceholder` would be better